src/legacy/core_plugins/kbn_vislib_vis_types/public/controls/point_series/value_axes.js

```diff
@@ -1,205 +0,0 @@
-          case 'top': return 'bottom';
-          case 'left': return 'right';
-          case 'right': return 'left';
-        }
-      }
-
-      $scope.rotateOptions = [
-        { name: 'horizontal', value: 0 },
-        { name: 'vertical', value: 90 },
-        { name: 'angled', value: 75 },
-      ];
-
-      $scope.$watch('editorState.params.categoryAxes[0].position', position => {
-        isCategoryAxisHorizontal = ['top', 'bottom'].includes(position);
-        $scope.editorState.params.valueAxes.forEach(axis => {
-          const axisIsHorizontal = ['top', 'bottom'].includes(axis.position);
-          if (axisIsHorizontal === isCategoryAxisHorizontal) {
-            axis.position = mapPosition(axis.position);
-            $scope.updateAxisName(axis);
-          }
-        });
-      });
-
-      $scope.getSeries = function (axis) {
-        const isFirst = $scope.editorState.params.valueAxes[0] === axis;
-        const series = $scope.editorState.params.seriesParams.filter(series =>
-          (series.valueAxis === axis.id || (isFirst && !series.valueAxis))
-        );
-        return series.map(series => series.data.label).join(', ');
-      };
-
-      $scope.getSeriesShort = function (axis) {
-        const maxStringLength = 30;
-        return $scope.getSeries(axis).substring(0, maxStringLength);
-      };
-
-      $scope.isPositionDisabled = function (position) {
-        if (isCategoryAxisHorizontal) {
-          return ['top', 'bottom'].includes(position);
-        }
-        return ['left', 'right'].includes(position);
-      };
-
-      $scope.addValueAxis = function () {
-        const firstAxis = $scope.editorState.params.valueAxes[0];
-        const newAxis = _.cloneDeep(firstAxis);
-        newAxis.id = 'ValueAxis-' + $scope.editorState.params.valueAxes.reduce((value, axis) => {
-          if (axis.id.substr(0, 10) === 'ValueAxis-') {
-            const num = parseInt(axis.id.substr(10));
-            if (num >= value) value = num + 1;
-          }
-          return value;
-        }, 1);
-
-        newAxis.position = mapPositionOpposite(firstAxis.position);
-        const axisName = _.capitalize(newAxis.position) + 'Axis-';
-        newAxis.name = axisName + $scope.editorState.params.valueAxes.reduce((value, axis) => {
-          if (axis.name.substr(0, axisName.length) === axisName) {
-            const num = parseInt(axis.name.substr(axisName.length));
-            if (num >= value) value = num + 1;
-          }
-          return value;
-        }, 1);
-
-        $scope.editorState.params.valueAxes.push(newAxis);
-        return newAxis;
-      };
-
-      $scope.removeValueAxis = function (axis) {
-        if ($scope.editorState.params.valueAxes.length > 1) {
-          _.remove($scope.editorState.params.valueAxes, function (valAxis) {
-            return valAxis.id === axis.id;
-          });
-        }
-      };
-
-      $scope.updateExtents = function (axis) {
-        if (!axis.scale.setYExtents) {
```

Comment:
note: moved to https://github.com/elastic/kibana/pull/43772/files?file-filters%5B%5D=.tsx&hide-deleted-files=true#diff-0d0929084cab92cbc7b2bba0204b8900R71